front/src/config/i18n/en.json

```diff
@@ -283,6 +321,11 @@
       "title": "MQTT",
       "deviceTab": "Devices",
       "setupTab": "Setup",
+      "status": {
+        "notConfigured": "MQTT service is not configured, please go to ",
+        "notConnected": "MQTT service failed to connect to network, please go to ",
```

Comment:
Please go to?

Reply:
To be able to use `preact-router Link`, the link is outside the i18n : 

![NotConnected](https://user-images.githubusercontent.com/1839717/67507656-a9245e80-f68f-11e9-8341-9f4f267af5fa.PNG)

Reply:
Oh my bad! Ok!